src/ui/server/routes/insurance/business/index.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import { ROUTES } from '../../../constants';
+
+import { get as getCompanyDetails, postCompaniesHouseSearch } from '../../../controllers/insurance/business/company-details';
+
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
In-line eslint rule?

Reply:
it flags express.Router so needs the rule, have removed the `/* eslint-disable @typescript-eslint/ban-ts-comment */` though